src/style.css

```diff
@@ -1,3 +1,91 @@
+::after {
+  --blue: #00acdc;
+}
+
 body {
+  margin: 0 auto;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 auto;
+
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.logo {
+  display: flex;
+  margin: 10px 50px;
+}
+
+.nav {
+  margin: 0 50px 0 0;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+  height: 60px;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+  flex-direction: row;
+
+}
+
+.is-active-item {
+  margin-left: 0;
+}
```

Comment:
please do not use `is-active-item` class, better to remove margin from first-child on `is-active` instead